app/assets/stylesheets/components/_organization_modal.scss

```diff
@@ -0,0 +1,29 @@
+.organization-modal {
+  background-color: $white;
+  left: 0;
+  margin: $base-spacing $base-spacing*4;
+  position: absolute;
+  right: 0;
+  top: 0;
+  z-index: 2;
+
+  &:before {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background-color: #000;
+    opacity: 0.5;
+    content: "";
+    z-index: -1;
+  }
+
+  &__contents {
+    background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.